updater.py

```diff
@@ -34,33 +36,84 @@ def buildFromSource(tag):
     # Move binary
     os.system("mv gitea "+settings.gtfile)
 
+def is_tool(name):
+    ##Check whether `name` is on PATH and marked as executable. 
+
+    # from whichcraft import which
+    from shutil import which
+
+    return which(name) is not None
+
+if not is_tool("xz"):
+	print ("missing dependency: xz")
+	quit()
+
 # Version from gitea site
 current_version = requests.get(settings.gtsite).json()['version']
-
+print ("current_version =", current_version)
 # Get version tag from github and remove first char (v)
 github_version_tag = requests.get(settings.gtgithubapiurl).json()['tag_name']
-
+print ("github_version_tag =", github_version_tag)
 # Get version from version tag
 github_version = github_version_tag[1:]
-
 # Check if there is a new version
 if github_version > current_version:
 
+
     # Stop systemd service
-    os.system("systemctl stop gitea.service")
+    print ("new version available, stopping service")
+    #os.system("systemctl stop gitea.service")
```

Comment:
Why is this line commented out?

Reply:
ahh. 
I used it on a live server, i didn't wanted to restart the service, a 1000 times.